### PR TITLE
Jing

### PR DESCRIPTION
**Pull request by JingtingL**

### CODE REVIEW

**Review by jut008 — APPROVED** [review]
lgtm

File: sleepbuddies/public/logo192.png
Comment (Shuzhengz) [inline sleepbuddies/public/logo192.png:1]: wait why are we adding this?

File: sleepbuddies/public/logo512.png
Comment (Shuzhengz) [inline sleepbuddies/public/logo512.png:1]: and this

File: client/src/App.tsx [hunk 1]
```diff
@@ -16,7 +19,9 @@ function App() {
               <h1>Sleep Buddies</h1>
           </header>
           <Routes>
-              <Route path="/" Component={Home}/>
+              <Route path="/" Component={welcome}/>
```
Comment (Shuzhengz) [inline client/src/App.tsx:22]: hm if we want to do a `remember me` checkbox then we shouldn't have the welcome page as the default page

File: client/src/components/elements/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    return (
+        <Link className="clickable" to="/login">
+            <div className="applet">
+                <img className="image" src={Image} alt={"Login icon"}/>
+            </div>
+        </Link>
+    );
+};
+
+export default Login;
```
Comment (Shuzhengz) [inline client/src/components/elements/Login.tsx:14]: @JingtingL should prob add newlines in the end of the files cuz of [this](https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3)

**Pull request merged**